packages/rum-recorder/src/boot/rumRecorderPublicApi.ts

```diff
@@ -31,7 +32,7 @@ export function makeRumRecorderPublicApi(startRumImpl: StartRum, startRecordingI
 
     const startRumResult = startRumImpl(userConfiguration, () => ({
       ...getCommonContext(),
-      hasReplay: state.status === RecorderStatus.Started ? true : undefined,
+      hasReplay: state.status === RecorderStatus.Started && state.isRecording() ? true : undefined,
```

Comment:
not clear to me what is the difference between state.status and isRecording 🤔
could we have a single source of truth? or make the difference clearer?